spec/views/shared/activities/activity_spec.rb

```diff
@@ -112,46 +116,89 @@
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.aid_type"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.fstc_applies"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_gender"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_climate_change_adaptation"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_climate_change_mitigation"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_biodiversity"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_desertification"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_disability"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.policy_marker_disaster_risk_reduction"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.covid19_related"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.channel_of_delivery_code"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.oda_eligibility"))
-            expect(rendered).not_to have_content(t("activerecord.attributes.activity.oda_eligibility_lead"))
+        context "when the activity has a linked activity" do
+          let(:activity) { create(factory_name, :ispf_funded, extending_organisation: user.organisation, organisation: user.organisation, ispf_theme: 1, ispf_partner_countries: ["IN"]) }
+
+          before do
+            activity.linked_activity = create(:third_party_project_activity, :ispf_funded)
+            activity.save
+
+            render
+          end
+
+          it "shows the linked activity" do
+            expect(rendered).to have_content(activity_presenter.linked_activity.title)
+          end
+
+          it "shows a link to edit the linked activity" do
+            expect(body.find(".linked_activity .govuk-summary-list__actions a")).to have_content(t("default.link.edit"))
           end
         end
       end
     end
+
+    context "when the activity is a project that has any child projects that are linked" do
+      let(:linked_proj) { create(:project_activity, :ispf_funded) }
+      let!(:third_party_project) { create(:third_party_project_activity, parent: activity, linked_activity: create(:third_party_project_activity, parent: linked_proj)) }
+      let(:activity) { create(:project_activity, :ispf_funded, linked_activity: linked_proj, organisation: user.organisation, extending_organisation: user.organisation) }
+
+      before { create(:report, :active, fund: activity.associated_fund, organisation: user.organisation) }
```

Comment:
Likewise is this preferable to putting it in a `let` or `let!`? If so, should it be combined with the next `before`?

Reply:
I think we might have discussed why this wouldn't be in a `let`? But it could probably still be combined with the `before` that follows